spec/linter-xmllint-spec.coffee

```diff
@@ -114,6 +114,19 @@ describe 'The xmllint provider for Linter', ->
           expect(messages.length).toEqual 1
           expect(messages[0].range).toEqual [[3, 2], [3, 21]]
     waitsForPromise ->
+      return atom.workspace.open(__dirname + '/fixtures/invalid/xml-model-unavailable.xml').then (editor) ->
+        return lint(editor).then (messages) ->
+          expect(messages.length).toBeGreaterThan 0
+          expect(messages.length).toBeLessThan 3
```

Comment:
Unfortunately I don't see anything that fits better in [the list of matchers](http://jasmine.github.io/1.3/introduction.html#section-Included_Matchers).